.pax/pre-packaging.sh

```diff
@@ -182,6 +182,47 @@ mv ./content/smpe  .
 echo "[$SCRIPT_NAME] templates is not part of zowe.pax, moving it out ..."
 mv ./content/templates  .
 
+# # . . . . . . . . . . . start of fingerprint . . . . . . . . . . . . . . . . . . . . . . .
+
+# This is the script where we create the fingerprint in the zowe.pax file.
+# To do this, we will create a runtime directory, somewhat like smpe.sh does today,
+# by running zowe-install.sh
+
+# For the moment, I will let smpe.sh continue to create its own runtime folder, but this creation
+# will be removed later for efficiency.
+# Note that smpe.sh is where we publish the fingerprint.  This should be moved back into here.
+
+# Note that zowe-install.sh creates USS files ... but also datasets, which need to be deleted.
+# TODO
+# To delete them, amend catchall-packaging.sh in this directory.
+
+# # Generate reference hash keys of runtime files
+echo "----- Generate reference hash keys of runtime files -----"
+echo Installing Zowe in temporary runtime directory
+mkdir zowe-runtime-dir 
+echo ROOT = $ROOT # do we have the ROOT to publish to?
+userid=${USER:-${USERNAME:-${LOGNAME}}}
+./content/zowe-$ZOWE_VERSION/install/zowe-install.sh -i zowe-runtime-dir -h $userid.TRUNTIME # [-l <log_directory>]
```

Comment:
Sorry to jump in because I'm not set as reviewer. But this is important and it's different from what we discussed, hope you don't mind I comment.

`-h $userid.TRUNTIME` has to be changed to a random dataset name like what we did in `smpe.sh`. If we use same name, two build jobs running at same time may overwrite the content of the others and will make the content of dataset not reliable.

Reply:
Will fix.